addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -213,7 +224,12 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
 
             const jsonResponse = response && JSON.parse(response);
             if (jsonResponse.success) {
+                const orderProgressBarEl = this.el.querySelector('.o_wevent_booth_order_progress');
+                orderProgressBarEl.parentElement.removeChild(orderProgressBarEl);
+                const boothCategoryId = this.el.querySelector('input[name=booth_category_id]').value;
```

Comment:
I guess it's easier to re-render the whole thing rather than just "updating" the existing one?

Would have been easier with OWL and re-active stuff :eyes: 

Also, isn't `orderProgressBarEl.parentElement.removeChild(orderProgressBarEl);` the same as juste `orderProgressBarEl.remove()` ?